master/getting-started/mesos/installation/dc-os/index.md

```diff
@@ -0,0 +1,119 @@
+
+**Alternatives: Installing Manually**
+
+The framework runs Calico (and its configuration) **within DC/OS.**
+This means it registers as a Mesos Framework, and uses Mesos Resource offers
+to run and configure the cluster with Calico. Alternative to this approach,
+Calico can be manually installed directly onto Agents as a daemon service integrated
+with the OS (using systemd) to ensure it is available when tasks are eventually
+provisioned.
+
+Next, we'll dive into each task the Framework performs.
+
+### etcd
+
+Calico uses etcd as its central database. There are two popular ways to run
+etcd in DC/OS:
+
+1. **Use the Universe etcd package**
+
+    The Universe etcd package launches a Mesos Framework that uses Mesos resource
+    offers to spawn a multi-node etcd cluster.
+    The endpoint endpoint address can be resolved via a SRV lookup of
+    `_etcd-server._tcp.etcd.mesos`.
+
+    Calico doesn't support connections to etcd via
+    SRV record, so the Calico-DC/OS Framework first runs its own instance
+    of etcd in proxy mode on every agent, which it relies on to forward requests
+    made to `localhost:2379` onwards to the actual etcd cluster.
+
+2. **Manually running etcd**
+
+    Running the etcd cluster manually across all masters can be considered more
+    stable than the previous option, as the endpoint address is static.
+    Users launching etcd in this way can skip running etcd in proxy mode, and
+    can simply change `ETCD_ENDPOINTS` to point directly at their static
+    etcd cluster.
+
+### Docker Cluster Store
+
+Calico networks Docker Containerizer tasks at the Docker-engine layer.
+To do multi-host networking in Docker, each docker engine must be configured
+to use the same cluster-store.
+
+By default, the Calico-DC/OS Framework will parse the value set for `ETCD_ENDPOINTS`,
+configure Docker to use it by adding it to `/etc/docker/daemon.json`,
+and finally restart Docker.
+
+Users can set `override-docker-cluster-store` to manually choose a different
+cluster store (e.g. the existing zookeeper on master), or, if they are only
+planning to use Calico for Unified Containerizer networking,
+can disable modification of the docker daemon altogether.
+
+### Calico CNI Installation
+
+To perform networking on Unified Containerizer tasks, Calico's CNI binaries and
+configuration file must be installed on every agent, and the slave process must
+be restarted to pick up the change. This step of the Framework performs the following steps:
+
+1. Download [`calico`](https://github.com/projectcalico/calico-cni/releases/latest) to `/opt/mesosphere/active/cni/`
+2. Download [`calico-ipam`](https://github.com/projectcalico/calico-cni/releases/latest) to `/opt/mesosphere/active/cni/`
+3. Create the following JSON file at `/opt/mesosphere/etc/dcos/network/cni/calico.cni`:
+
+   ```json
+   {
+       "name": "calico",
+       "type": "calico",
+       "etcd_endpoints": "http://m1.dcos:2379,m2.dcos:2379",
```

Comment:
Is this the best suggestion for how to access etcd?  Given the suggestion above to run etcd in proxy mode.  Maybe just add a sentence:

"If running etcd in proxy mode, use `http://localhost:2379` for etcd_endpoints.  Otherwise, update the field to point to your etcd cluster."